batavia/VirtualMachine.js

```diff
@@ -9,7 +9,21 @@ batavia.VirtualMachine = function(loader) {
 
     if (loader === undefined) {
         this.loader = function(name) {
-            return document.getElementById('batavia-' + name).text.replace(/(\r\n|\n|\r)/gm, "").trim();
+            var element = document.getElementById('batavia-' + name);
```

Comment:
It's finding the `<script>` block that contains base64 encoded bytecode, getting the text contents of that block, and stripping out all the whitespace. That content is the payload that can then be decoded from base64 and passed to the marshaller.

I'll add this as a comment in the code for posterity.